src/js/components/dashboard/summary/DashboardSummary.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+  * DashboardSummary.jsx
+  * Created by Daniel Boos 11/12/19
+  */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import DashboardSummaryRow from './DashboardSummaryRow';
+
+const propTypes = {
+    file: PropTypes.string,
+    agency: PropTypes.string,
+    submissions: PropTypes.array
+};
+
+function getSummaryRows(submissions, file) {
+    const summaryRows = [];
+    let key = 0;
+    if (submissions.length === 0) {
+        summaryRows.push(<DashboardSummaryRow key={key} />);
+    }
+    else {
+        submissions.forEach((submission) => {
```

Comment:
You could also use [Array.prototype.map()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) here